### ISSUE TITLE
NTR: Ultrasound or MRI: jejunal duplication (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8992]
New term request Ultrasound or MRI: jejunal duplication (elthat_110428034721):

**Comment by pnrobinson** [comment]
@MickeySegal
are we talking about jejunal duplication cyst?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9289338/

**Comment by MickeySegal** [comment]
Our citation is in Mosaic Trisomy 8.  In Smith's Malformation textbook
"jejunal duplication"

Also described in https://pubmed.ncbi.nlm.nih.gov/1773546/ "Mosaic trisomy 8 associated with jejunal duplication"

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6001035
- label: Tubular jejunal duplication
- parents: [Abnormal jejunum morphology (HP:0005265)](https://hpo.jax.org/app/browse/term/HP:0005265)
- pmid: PMID:1773546
- def: A segment of the jejunum that runs parallel to the normal jejunum and communicates with the jejunal lumen at one or both ends.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8992
- synonyms:
